css/selectors/active-view-transition-type.json

```diff
@@ -0,0 +1,44 @@
+{
+  "css": {
+    "selectors": {
+      "active-view-transition-type": {
+        "__compat": {
+          "description": "<code>:active-view-transition-type()</code>",
+          "spec_url": "https://drafts.csswg.org/css-view-transitions-2/#the-active-view-transition-type-pseudo",
+          "tags": [
+            "web-features:view-transitions"
```

Comment:
@foolip I'm assuming you want a new feature tag for these? `"web-features:view-transition-types"`? Advise appreciated.

Reply:
I would suggest just not tagging these new features for now and making that decision later.